### DISCUSSION TITLE
Non-openAI embedding models for better performance

### DISCUSSION THREAD

**Discussion by dougkeiller** [brianpetro/obsidian-smart-connections#604] [Ideas]
While many new local embedding models have been added, on my laptop I find local embeddings slow. I suggest adding more cloud embedding models in addition to openAI.

This ranking of embedding model performance on Hugging Face could help select the next service to add. We should look at th Retrieval column as the key metric, per https://www.linkedin.com/pulse/enhance-your-rag-llm-performance-discovering-perfect-embedding-eezfc/

https://huggingface.co/spaces/mteb/leaderboard

Screenshot of the embedding models ranked by retrieval:

![image](https://github.com/brianpetro/obsidian-smart-connections/assets/128188043/9918749b-ff2b-492f-bcd7-64f8e414d20c)

**Comment by hittjwX** [comment]
A local model helps narrow the scope of training to your authoritative notes. If working on research or proprietary notes, I wouldn't want to risk bringing in outside data without vetting or having data leaking.
